pep-0694.rst

```diff
@@ -39,9 +39,9 @@ Beyond the above, there are a number of major issues with the current API:
   from source.
 
 - It has very limited support for communicating back to the user, limited entirely
-  to the HTTP status code, and status message (something which I'm not sure is
-  even technically valid HTTP?). It has no support for multiple errors, warnings,
-  deprecations, etc.
+  to the HTTP status code and reason phrase, which has been deprecated 
```

Comment:
This is ambiguous -- is the status code also deprecated, or only the reason phrase?

Reply:
Just the reason phrase, which the status quo file upload API on PyPI uses custom reason phrases to communicate errors back to the user. It would be good to make it less ambiguous though.

Reply:
Yeah, it definitely should be less ambiguous. As a starting point, what about

```suggestion
  to the HTTP status code and the (deprecated) reason phrase,
```